packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -115,24 +115,6 @@ export default class ConfigParser {
             this._config.exclude = this.setFilePathToFilterOptions(exclude, this._config.exclude!)
         }
 
-        /**
```

Comment:
Don't remember what this code do, I guess you removed it for a reason

Reply:
yes, needed to increase test coverage and this wasn't covered by any tests and the bug that was fixed with it doesn't rely on this piece of code. Haven't seen any reason to keep it around.